bob-apps/bin/bobc.rs

```diff
@@ -1,13 +1,22 @@
 #[macro_use]
 extern crate log;
 
+#[macro_use]
+extern crate lazy_static;
+
 use bob::{Blob, BlobKey, BlobMeta, BobApiClient, GetRequest, PutRequest};
 use clap::{App, Arg, ArgMatches, SubCommand};
 use http::Uri;
 use log::LevelFilter;
 use std::time::{SystemTime, UNIX_EPOCH};
 use tonic::Request;
 
+lazy_static! {
+    static ref KEY_SIZE: usize = option_env!("BOB_KEY_SIZE")
```

Comment:
It is better to read `key_size` from command line arguments (like it is done in bobp)